lib/edge/tools/s3_proxy/s3_proxy.sh

```
@@ -0,0 +1,168 @@
+  fi
+  log "starting s3proxy ($BUCKET <- $DATA_DIR) on $ENDPOINT"
+  docker run -d --name "$CONTAINER" --cap-add NET_ADMIN \
+    -p "${PORT}:80" \
+    -v "$DATA_DIR:/data" \
+    -e S3PROXY_AUTHORIZATION=none \
+    -e S3PROXY_ENDPOINT=http://0.0.0.0:80 \
+    -e JCLOUDS_PROVIDER=filesystem \
+    -e JCLOUDS_FILESYSTEM_BASEDIR=/data \
+    "$S3PROXY_IMAGE" >/dev/null
+  local i code
+  for i in $(seq 1 30); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)